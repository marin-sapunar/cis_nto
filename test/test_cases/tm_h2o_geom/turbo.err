 dscf ended normally
 ricc2 ended normally
