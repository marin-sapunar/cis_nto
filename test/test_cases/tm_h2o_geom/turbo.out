 operating system is UNIX !
 hostname is         neptun

 dscf (neptun) : TURBOMOLE V7.0.1( 19978 ) 29 Sep 2015 at 10:17:19
 Copyright (C) 2015 TURBOMOLE GmbH, Karlsruhe


    2019-06-10 17:13:31.813 



                             d s c f - program

                   idea & directorship : reinhart ahlrichs
                   program development : marco haeser
                                         michael baer
                           dft version : oliver treutler


                          quantum chemistry group
                          universitaet  karlsruhe
                                  germany




                          References             
                                                 
          TURBOMOLE:                             
              R. Ahlrichs, M. Baer, M. Haeser, H. Horn, and
              C. Koelmel
              Electronic structure calculations on workstation
              computers: the program system TURBOMOLE
              Chem. Phys. Lett. 162: 165 (1989)
          Density Functional:                              
              O. Treutler and R. Ahlrichs                      
              Efficient Molecular Numerical Integration Schemes
              J. Chem. Phys. 102: 346 (1995)                   
          Parallel Version:                                
              Performance of parallel TURBOMOLE for Density    
              Functional Calculations                          
              M. v. Arnim and R. Ahlrichs                      
              J. Comp. Chem. 19: 1746 (1998)                   
                                         




              +--------------------------------------------------+
              | Atomic coordinate, charge and isotop information |
              +--------------------------------------------------+

                    atomic coordinates            atom    charge  isotop
          0.10000000    0.00000000   -0.12178984    o      8.000     0
          1.41713421    0.00000000    0.96657855    h      1.000     0
         -1.41713421    0.00000000    0.96657855    h      1.000     0
 
       center of nuclear mass  :    0.08880987    0.00000000   -0.00000000
       center of nuclear charge:    0.08000000    0.00000000    0.09588384

              +--------------------------------------------------+
              |               basis set information              |
              +--------------------------------------------------+

              we will work with the 1s 3p 5d 7f 9g ... basis set
              ...i.e. with spherical basis functions...

   type   atoms  prim   cont   basis
   ---------------------------------------------------------------------------
    o        1     24     14   def-SV(P)   [3s2p1d|7s4p1d]
    h        2      4      2   def-SV(P)   [2s|4s]
   ---------------------------------------------------------------------------
   total:    3     32     18
   ---------------------------------------------------------------------------

   total number of primitive shells          :   16
   total number of contracted shells         :   10
   total number of cartesian basis functions :   19
   total number of SCF-basis functions       :   18


 integral neglect threshold       :  0.18E-09
 integral storage threshold THIZE :  0.10E-04
 integral storage threshold THIME :         5


 DENSITY CONVERGENCE CHECK SWITCHED ON !
 SCF CONVERGENCE IF RMS(delta[D]) <     0.1000000000E-06


 symmetry group of the molecule :   c1 

 the group has the following generators :
   c1(z)

    1 symmetry operations found

 there are 1 real representations :   a   

 maximum number of shells which are related by symmetry :  1


    mo occupation :
   irrep   mo's   occupied
    a       18        5
 
 number of basis functions   :           18
 number of occupied orbitals :            5
 

          ------------------------
          nuclear repulsion energy  :   9.31957038014    
          ------------------------

 
          -----------------
          -S,T+V- integrals
          -----------------

 1e-integrals will be neglected if expon. factor < 0.175893E-10
 
   Difference densities algorithm switched on.
   The maximal number of linear combinations of
   difference densities is          20 .

 automatic virtual orbital shift switched on 
      shift if e(lumo)-e(homo) < 0.10000000    


 DIIS switched on: error vector is FDS-SDF
 Max. Iterations for DIIS is     :   5
 DIIS matrix (see manual) 
    Scaling factor of diagonals  :  1.200
    threshold for scaling factor :  0.000

 scf convergence criterion : increment of total energy < .1000000D-06
                  and increment of one-electron energy < .1000000D-03

  MOs are in ASCII format !


 reading orbital data $scfmo  from file mos . 

 orbital characterization : scfconv=7
 

 DSCF restart information will be dumped onto file mos

 
                                              current damping :  0.300
 ITERATION  ENERGY          1e-ENERGY        2e-ENERGY     RMS[dD(SAO)]   TOL
   1  -75.932877962736    -123.17277133     37.920322986    0.000D+00 0.175D-09
          max. resid. norm for Fia-block=  2.436D-02 for orbital      3a         
          max. resid. fock norm         =  5.763D-02 for orbital      8a         
          Delta Eig. =     0.3426305548 eV 
 
                                              current damping :  0.250
 ITERATION  ENERGY          1e-ENERGY        2e-ENERGY     RMS[dD(SAO)]   TOL
   2  -75.934181135674    -123.18042640     37.926674885    0.339D-02 0.111D-09
          Norm of current diis error: 0.34100E-01
          max. resid. norm for Fia-block=  1.259D-02 for orbital      3a         
          max. resid. fock norm         =  1.507D-02 for orbital      8a         
          Delta Eig. =     0.1727699833 eV 
 
                                              current damping :  0.200
 ITERATION  ENERGY          1e-ENERGY        2e-ENERGY     RMS[dD(SAO)]   TOL
   3  -75.934441184800    -123.17690115     37.922889589    0.150D-02 0.859D-10
          Norm of current diis error: 0.12175E-01
          max. resid. norm for Fia-block=  5.144D-03 for orbital      3a         
          max. resid. fock norm         =  5.491D-03 for orbital      3a         
          Delta Eig. =     0.0374821684 eV 
 
                                              current damping :  0.250
 ITERATION  ENERGY          1e-ENERGY        2e-ENERGY     RMS[dD(SAO)]   TOL
   4  -75.934479287139    -123.17865604     37.924606373    0.584D-03 0.737D-10
          Norm of current diis error: 0.33857E-02
          max. resid. norm for Fia-block=  1.501D-03 for orbital      3a         
          max. resid. fock norm         =  1.649D-03 for orbital      3a         
          Delta Eig. =     0.0101264031 eV 
 
                                              current damping :  0.300
 ITERATION  ENERGY          1e-ENERGY        2e-ENERGY     RMS[dD(SAO)]   TOL
   5  -75.934482512056    -123.17747605     37.923423161    0.165D-03 0.683D-10
          Norm of current diis error: 0.10079E-02
          max. resid. norm for Fia-block=  4.524D-04 for orbital      3a         
          max. resid. fock norm         =  5.092D-04 for orbital      3a         
 mo-orthogonalization: Cholesky decomposition
          Delta Eig. =     0.0048253145 eV 
 
                                              current damping :  0.350
 ITERATION  ENERGY          1e-ENERGY        2e-ENERGY     RMS[dD(SAO)]   TOL
   6  -75.934482827106    -123.17785633     37.923803124    0.526D-04 0.653D-10
          Norm of current diis error: 0.32684E-03
          max. resid. norm for Fia-block=  1.448D-04 for orbital      3a         
          max. resid. fock norm         =  1.649D-04 for orbital      3a         
          Delta Eig. =     0.0015653256 eV 
 
                                              current damping :  0.400
 ITERATION  ENERGY          1e-ENERGY        2e-ENERGY     RMS[dD(SAO)]   TOL
   7  -75.934482861929    -123.17768158     37.923628333    0.184D-04 0.626D-10
          Norm of current diis error: 0.11308E-03
          max. resid. norm for Fia-block=  4.831D-05 for orbital      3a         
          max. resid. fock norm         =  5.668D-05 for orbital      3a         
          Delta Eig. =     0.0004902764 eV 
 
                                              current damping :  0.450
 ITERATION  ENERGY          1e-ENERGY        2e-ENERGY     RMS[dD(SAO)]   TOL
   8  -75.934482866386    -123.17771777     37.923664519    0.681D-05 0.596D-10
          Norm of current diis error: 0.42061E-04
          max. resid. norm for Fia-block=  1.674D-05 for orbital      3a         
          max. resid. fock norm         =  2.011D-05 for orbital      3a         
          Delta Eig. =     0.0000993225 eV 
 
                                              current damping :  0.500
 ITERATION  ENERGY          1e-ENERGY        2e-ENERGY     RMS[dD(SAO)]   TOL
   9  -75.934482867396    -123.17770449     37.923651243    0.260D-05 0.563D-10
          Norm of current diis error: 0.16730E-04
          max. resid. norm for Fia-block=  7.623D-06 for orbital      4a         
          max. resid. fock norm         =  8.399D-06 for orbital      4a         
          Delta Eig. =     0.0000292804 eV 

 ENERGY & DENSITY CONVERGED !

 
                                              current damping :  0.550
 ITERATION  ENERGY          1e-ENERGY        2e-ENERGY     RMS[dD(SAO)]   TOL
  10  -75.934482868023    -123.17770538     37.923652130    0.106D-05 0.530D-10
          max. resid. norm for Fia-block=  2.492D-06 for orbital      4a         
          max. resid. fock norm         =  2.678D-06 for orbital      4a         

 convergence criteria satisfied after 10 iterations


                  ------------------------------------------ 
                 |  total energy      =    -75.93448286802  |
                  ------------------------------------------ 
                 :  kinetic energy    =     75.94537298690  :
                 :  potential energy  =   -151.87985585492  :
                 :  virial theorem    =      2.00014341467  :
                 :  wavefunction norm =      1.00000000000  :
                  .......................................... 


 orbitals $scfmo  will be written to file mos

    irrep                  1a          2a          3a          4a          5a   
 eigenvalues H        -20.54004    -1.33228    -0.71754    -0.57217    -0.50317
            eV        -558.9274    -36.2535    -19.5253    -15.5697    -13.6921
 occupation              2.0000      2.0000      2.0000      2.0000      2.0000 

    irrep                  6a          7a          8a          9a         10a   
 eigenvalues H          0.17402     0.25568     0.80496     0.96788     1.19833
            eV           4.7354      6.9575     21.9043     26.3374     32.6086
 
 
 
 
 ==============================================================================
                           electrostatic moments
 ==============================================================================

 reference point for electrostatic moments:    0.00000   0.00000   0.00000

 
              nuc           elec       ->  total
 ------------------------------------------------------------------------------
                          charge      
 ------------------------------------------------------------------------------
          10.000000     -10.000000      -0.000000
 
 ------------------------------------------------------------------------------
                       dipole moment  
 ------------------------------------------------------------------------------
   x       0.800000      -0.856000      -0.056000
   y       0.000000      -0.000000      -0.000000
   z       0.958838      -0.070439       0.888399
 
   | dipole moment | =     0.8902 a.u. =     2.2626 debye 
 
 ------------------------------------------------------------------------------
                     quadrupole moment
 ------------------------------------------------------------------------------
  xx       4.096539      -7.248303      -3.151765
  yy       0.000000      -5.294956      -5.294956
  zz       1.987210      -6.296221      -4.309011
  xy       0.000000       0.000000       0.000000
  xz      -0.097432       0.126447       0.029015
  yz       0.000000       0.000000       0.000000
 
     1/3  trace=      -4.251911
     anisotropy=       1.858713
 
 ==============================================================================
 
 


    ------------------------------------------------------------------------
         total  cpu-time :   0.04 seconds
         total wall-time :   0.04 seconds
    ------------------------------------------------------------------------


   ****  dscf : all done  ****


    2019-06-10 17:13:31.857 

 hostname is         neptun
 this is node-proc. number 0 running on node neptun
 the total number of node-proc. spawned is   1

 ricc2 (neptun) : TURBOMOLE V7.0.1( 19978 ) 29 Sep 2015 at 10:17:19
 Copyright (C) 2015 TURBOMOLE GmbH, Karlsruhe


    2019-06-10 17:13:33.742 



                              R I C C 2 - PROGRAM

                          the quantum chemistry groups
                             at the universities in
                           Karlsruhe, Bochum & Mainz
                                   Germany



   *-----------------------------------------------------------------------*
   |                     program will use  1 thread(s)                     |
   *-----------------------------------------------------------------------*


              +--------------------------------------------------+
              | Atomic coordinate, charge and isotop information |
              +--------------------------------------------------+

                    atomic coordinates            atom    charge  isotop
          0.10000000    0.00000000   -0.12178984    o      8.000     0
          1.41713421    0.00000000    0.96657855    h      1.000     0
         -1.41713421    0.00000000    0.96657855    h      1.000     0
 
       center of nuclear mass  :    0.08880987    0.00000000   -0.00000000
       center of nuclear charge:    0.08000000    0.00000000    0.09588384

              +--------------------------------------------------+
              |               basis set information              |
              +--------------------------------------------------+

              we will work with the 1s 3p 5d 7f 9g ... basis set
              ...i.e. with spherical basis functions...

   type   atoms  prim   cont   basis
   ---------------------------------------------------------------------------
    o        1     24     14   def-SV(P)   [3s2p1d|7s4p1d]
    h        2      4      2   def-SV(P)   [2s|4s]
   ---------------------------------------------------------------------------
   total:    3     32     18
   ---------------------------------------------------------------------------

   total number of primitive shells          :   16
   total number of contracted shells         :   10
   total number of cartesian basis functions :   19
   total number of SCF-basis functions       :   18



   =========================================================================


     restricted closed shell calculation for the wavefunction models:
               ADC(2)     - Algebraic Diagrammatic Construction to 2. Order (strict)


     global parameters for ricc2 program:

        hard restart (reuse of interm.) :  enabled 
        soft restart (reuse of vectors) :  disabled
        threshold for vector function   :    0.100000E-05
        convergence threshold energy    :    0.100000E-06
        linear dependence threshold     :    0.100000E-13
        global print level              :      1
        maximum number of iterations    :    30
        maximum number DIIS vectors     :    10
        max. dim. of reduced space      :   100
        core memory limit (MB)          :   500


   =========================================================================
     "cartesian on" in $drvopt is ignored by $ricc2

    der. integral neglect threshold  :  0.10E-07
    integral neglect threshold       :  0.18E-09
    integral storage threshold THIZE :  0.10E-04
    integral storage threshold THIME :         5


  ----------------------------------------------------------------------------
  hard_restart flag is disabled because the check sum for the
  nuclear repulsion "$nucrep" differs from the reference value:
  reference value:     9.30715477357722    
  computed value :     9.31957038014281    
  relative error :    0.133398518318763E-02
  tolerance      :    0.100000000000000E-11
  the calculation will proceed...
  ----------------------------------------------------------------------------

                  +------------------------------------------+
                  |     Auxiliary basis set information      |
                  +------------------------------------------+

              we will work with the 1s 3p 5d 7f 9g ... basis set
              ...i.e. with spherical basis functions...

   type   atoms  prim   cont   basis
   ---------------------------------------------------------------------------
    o        1     72     48   def-SV(P)   [6s5p4d1f|8s6p5d3f]
    h        2     23     14   def-SV(P)   [3s2p1d|4s3p2d]
   ---------------------------------------------------------------------------
   total:    3    118     76
   ---------------------------------------------------------------------------

   total number of primitive shells          :   31
   total number of contracted shells         :   28
   total number of cartesian basis functions :   85
   total number of SCF-basis functions       :   76


 symmetry group of the molecule :   c1 

 the group has the following generators :
   c1(z)

    1 symmetry operations found

 there are 1 real representations :   a   

 maximum number of shells which are related by symmetry :  1


 The symmetry information takes   1 MB

*---------------------------------------------------------------------*
|               simplified C1 algorithm will be applied               |
*---------------------------------------------------------------------*
  MOs are in ASCII format !


 reading orbital data $scfmo  from file mos . 

 orbital characterization : scfconv=7
 

         --------------------------------------------------------------
                   checking mo's by calculating trace(D*S)

              number of electrons : 
                 from trace(D*S)         :   10.000000000003
                 from occupation numbers :   10.000000000000

              molecular charge           :    0.000000000000
         --------------------------------------------------------------

 time elapsed for calculating density matrices :   0.000 sec
 all orbitals will be active in the correlation treatment


   Molecular Orbital Statistic:
   ============================

   -----------------------------
   orbitals in total:
   -----------------------------
    frozen occupied :      0
    active occupied :      5
    active virtual  :     13
    frozen virtual  :      0
    all together    :     18
   -----------------------------


     time in riccmos       cpu:  0.00 sec    wall:  0.00 sec    ratio:  1.2

     total memory allocated for calculation of (Q|P)**(-1/2) : 1 MiB


     calculation of (P|Q) ...
     time in lp2sym        cpu:  0.00 sec    wall:  0.00 sec    ratio:  0.0


     calculation of the Cholesky decomposition of (P|Q)**(-1) ...
     time in invmetri      cpu:  0.00 sec    wall:  0.00 sec    ratio:  0.6

   threshold for RMS(d[D]) in SCF was     :  0.10E-06
   integral neglect threshold             :  0.18E-09
   derivative integral neglect threshold  :  0.10E-07


 setting up bound for integral derivative estimation

 increment for numerical differentiation : 0.00050000

   =========================================================================

     Energy of reference wave function is   -75.9344828680200
     Maximum orbital residual is           0.2678118255463E-05


     Number of symmetry-nonredundant auxiliary basis functions:       76

     Block lengths for integral files:
        frozen occupied (BOI):        0 MiB
        active occupied (BJI):        1 MiB
        active virtual  (BAI):        1 MiB
        frozen virtual  (BGI):        0 MiB
               general  (BTI):        1 MiB

   =========================================================================




   **************************************************************************
   *                                                                        *
   *          OPTIMIZATION OF THE GROUND STATE CLUSTER AMPLITUDES           *
   *                                                                        *
   **************************************************************************


   Iter.   MP2 energy       Norm(Omega)    Norm(t1)  Norm(t2)     cpu    wall
   ...........................................................................
    1    -76.1236114435    0.0255498256     0.00000   0.12194    0.00    0.00
   ...........................................................................


     time in total         cpu:  0.00 sec    wall:  0.00 sec    ratio:  1.0


     **********************************************************************
     *                                                                    *
     *   RHF  energy                             :    -75.9344828680      *
     *   MP2 correlation energy                  :     -0.1891285755      *
     *                                                                    *
     *   Final MP2 energy                        :    -76.1236114435      *
     *                                                                    *
     *   Norm of MP1 T2 amplitudes               :      0.1219381291      *
     *                                                                    *
     *   D1 diagnostic                           :      0.0089            *
     *                                                                    *
     **********************************************************************



 **************************************************************
 *                                                            *
 *    OPTIMIZATION OF THE EXCITED STATE CLUSTER AMPLITUDES    *
 *                                                            *
 **************************************************************


  preoptimize excitation energies/vectors using CCS:
  ==================================================


 ======== LINEAR CC RESPONSE SOLVER ========

    threshold for convergence: 0.10E-02
    maximum number of simultaneous jacoby matrix transformations:  5
                                                
     summary of start vectors generation:
     -------------------------------------------
     type of solution vectors      : RE0
     symmetry                      : a  
     number of vectors requested   :     5
     number of vectors generated   :     5
       -- guessed as rhs/(e_a-e_i) :     5
     -------------------------------------------
                                                

    Iter  #Vectors  time (min)    max. residual 
    --------------------------------------------
      1       5         0.00     0.13E+00  (  3)
      2       5         0.00     0.44E-01  (  3)
      3       5         0.00     0.68E-02  (  3)
      4       5         0.00     0.20E-02  (  3)
      5       1         0.00     0.88E-03  (  4)
    --------------------------------------------
    converged in  5 iterations

         Total time          0.00 minutes


 ===========================================




 Note that this is the intermediate result of a preoptimization!
 The convergence threshold was  0.1E-02 only!


 +==================================================================+
 | sym | multi | state |          CCS excitation energies           |
 |     |       |       +--------------------------------------------+
 |     |       |       |   Hartree    |     eV       |     cm-1     |
 +==================================================================+
 | a   |   1   |   1   |    0.3419550 |      9.30507 |    75050.442 |
 | a   |   1   |   2   |    0.4150363 |     11.29371 |    91089.937 |
 | a   |   1   |   3   |    0.4368219 |     11.88653 |    95871.323 |
 | a   |   1   |   4   |    0.5083035 |     13.83164 |   111559.727 |
 | a   |   1   |   5   |    0.5659778 |     15.40104 |   124217.768 |
 +==================================================================+




  preoptimize ADC(2) excitation energies/vectors using a quasi linear algorithm:
  ==============================================================================


 ======== LINEAR CC RESPONSE SOLVER ========

    threshold for convergence: 0.10E-02
    maximum number of simultaneous jacoby matrix transformations:  5
                                                
     summary of start vectors generation:
     -------------------------------------------
     type of solution vectors      : RE0
     symmetry                      : a  
     number of vectors requested   :     5
     number of vectors generated   :     5
       -- restarted from file      :     5
     -------------------------------------------
                                                

    Iter  #Vectors  time (min)    max. residual 
    --------------------------------------------
 threshold for this macro iteration:  4.198457620634360E-002
      1       5         0.00     0.53E-01  (  2)
 ...adapted threshold to:  4.498085990601713E-002
      2       4         0.00     0.34E-01  (  5)
                                                
     summary of start vectors generation:
     -------------------------------------------
     type of solution vectors      : RE0
     symmetry                      : a  
     number of vectors requested   :     5
     number of vectors generated   :     5
       -- restarted from file      :     5
     -------------------------------------------
                                                

    Iter  #Vectors  time (min)    max. residual 
    --------------------------------------------
 threshold for this macro iteration:  2.222242509235584E-003
      3       5         0.00     0.34E-01  (  5)
      4       5         0.00     0.37E-02  (  5)
      5       1         0.00     0.20E-02  (  3)
                                                
     summary of start vectors generation:
     -------------------------------------------
     type of solution vectors      : RE0
     symmetry                      : a  
     number of vectors requested   :     5
     number of vectors generated   :     5
       -- restarted from file      :     5
     -------------------------------------------
                                                

    Iter  #Vectors  time (min)    max. residual 
    --------------------------------------------
 threshold for this macro iteration:  1.000000000000000E-003
      6       5         0.00     0.53E-03  (  3)
    --------------------------------------------
    converged in  6 iterations

         Total time          0.00 minutes


 ===========================================




 Note that this is the intermediate result of a preoptimization!
 The convergence threshold was  0.1E-02 only!


 +==================================================================+
 | sym | multi | state |          ADC(2) excitation energies        |
 |     |       |       +--------------------------------------------+
 |     |       |       |   Hartree    |     eV       |     cm-1     |
 +==================================================================+
 | a   |   1   |   1   |    0.2996411 |      8.15365 |    65763.610 |
 | a   |   1   |   2   |    0.3818262 |     10.39002 |    83801.157 |
 | a   |   1   |   3   |    0.3938853 |     10.71817 |    86447.840 |
 | a   |   1   |   4   |    0.4772347 |     12.98622 |   104740.901 |
 | a   |   1   |   5   |    0.5547090 |     15.09440 |   121744.557 |
 +==================================================================+





  solve for ADC(2) excitation energies/vectors using a DIIS algorithm:
  =================================================================
                                                
     summary of start vectors generation:
     -------------------------------------------
     type of solution vectors      : RE0
     symmetry                      : a  
     number of vectors requested   :     5
     number of vectors generated   :     5
       -- restarted from file      :     5
     -------------------------------------------
                                                


 ========= DIIS CC RESPONSE SOLVER =========

    threshold for convergence:0.10E-05
    max. nb. of DIIS vectors :      10
    maximum number of simultaneous jacoby matrix transformations:  5

    Iter  #Vectors  time (min)   residual
    --------------------------------------
      1       5         0.00     0.19E-03  (  5)    0.62E-05
      2       5         0.00     0.12E-03  (  5)    0.55E-05
      3       5         0.00     0.37E-04  (  5)    0.55E-05
      4       5         0.00     0.44E-05  (  5)    0.55E-05
      5       3         0.00     0.50E-06  (  5)    0.55E-05
    --------------------------------------
    converged in   5 iterations

   Total time          0.00 minutes


 ===========================================





 +================================================================================+
 | sym | multi | state |          ADC(2) excitation energies    |  %t1   |  %t2   |
 |     |       |       +----------------------------------------+--------+--------+
 |     |       |       |   Hartree    |    eV      |    cm-1    |    %   |    %   |
 +================================================================================+
 | a   |   1   |   1   |    0.2996461 |    8.15378 |  65764.707 |  95.29 |   4.71 |
 | a   |   1   |   2   |    0.3818290 |   10.39009 |  83801.769 |  95.60 |   4.40 |
 | a   |   1   |   3   |    0.3938900 |   10.71829 |  86448.853 |  95.24 |   4.76 |
 | a   |   1   |   4   |    0.4772369 |   12.98628 | 104741.392 |  95.67 |   4.33 |
 | a   |   1   |   5   |    0.5547071 |   15.09435 | 121744.146 |  96.67 |   3.33 |
 +================================================================================+


       Energy:     0.2996461 H      8.15378 eV    65764.707 cm-1

     +=======================================================================+
     | type: RE0                    symmetry: a               state:    1    |
     +-----------------------+-----------------------+-----------------------+
     | occ. orb.  index spin | vir. orb.  index spin |  coeff/|amp|     %    |
     +=======================+=======================+=======================+
     |    5 a        5       |    6 a        6       |   0.99457      98.9   |
     +=======================+=======================+=======================+
     norm of printed elements:  0.98917


       Energy:     0.3818290 H     10.39009 eV    83801.769 cm-1

     +=======================================================================+
     | type: RE0                    symmetry: a               state:    2    |
     +-----------------------+-----------------------+-----------------------+
     | occ. orb.  index spin | vir. orb.  index spin |  coeff/|amp|     %    |
     +=======================+=======================+=======================+
     |    5 a        5       |    7 a        7       |   0.98968      97.9   |
     +=======================+=======================+=======================+
     norm of printed elements:  0.97946


       Energy:     0.3938900 H     10.71829 eV    86448.853 cm-1

     +=======================================================================+
     | type: RE0                    symmetry: a               state:    3    |
     +-----------------------+-----------------------+-----------------------+
     | occ. orb.  index spin | vir. orb.  index spin |  coeff/|amp|     %    |
     +=======================+=======================+=======================+
     |    4 a        4       |    6 a        6       |   0.99107      98.2   |
     +=======================+=======================+=======================+
     norm of printed elements:  0.98222


       Energy:     0.4772369 H     12.98628 eV   104741.392 cm-1

     +=======================================================================+
     | type: RE0                    symmetry: a               state:    4    |
     +-----------------------+-----------------------+-----------------------+
     | occ. orb.  index spin | vir. orb.  index spin |  coeff/|amp|     %    |
     +=======================+=======================+=======================+
     |    4 a        4       |    7 a        7       |   0.99216      98.4   |
     +=======================+=======================+=======================+
     norm of printed elements:  0.98439


       Energy:     0.5547071 H     15.09435 eV   121744.146 cm-1

     +=======================================================================+
     | type: RE0                    symmetry: a               state:    5    |
     +-----------------------+-----------------------+-----------------------+
     | occ. orb.  index spin | vir. orb.  index spin |  coeff/|amp|     %    |
     +=======================+=======================+=======================+
     |    3 a        3       |    6 a        6       |   0.99292      98.6   |
     +=======================+=======================+=======================+
     norm of printed elements:  0.98590



      The semi-canonical algorithm will be used for densities


    ------------------------------------------------------------------------
         total  cpu-time :   0.20 seconds
         total wall-time :   0.21 seconds
    ------------------------------------------------------------------------


   ****  ricc2 : all done  ****


    2019-06-10 17:13:33.946 

